orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go

```diff
@@ -59,6 +59,55 @@ func (store *sqlConfiguratorStorage) loadFromEntitiesTable(networkID string, fil
 	return entsByPk, nil
 }
 
+func (store *sqlConfiguratorStorage) countEntitiesTable(networkID string, filter EntityLoadFilter, criteria EntityLoadCriteria) (uint64, error) {
+	selectBuilder, err := store.getEntitiesCountBuilder(networkID, filter, criteria)
+	if err != nil {
+		return 0, err
+	}
+	var count uint64
+	if err = selectBuilder.RunWith(store.tx).Scan(&count); err != nil {
+		return 0, err
+	}
+	return count, nil
+}
+
+func (store *sqlConfiguratorStorage) getEntitiesCountBuilder(networkID string, filter EntityLoadFilter, criteria EntityLoadCriteria) (sq.SelectBuilder, error) {
```

Comment:
This is duplicating a lot of logic. It's basically the same as the normal builder, but with "select count" instead of "select rows". I think there's a way to work those together

That said, wait for this PR until we get the configurator changes merged. Cause it will change things enough that you'll need to rework just a bit